Module1/ansible/hosts

```diff
@@ -0,0 +1,2 @@
+[module4]
+mod4node ansible_host=35.179.70.200
```

Comment:
I'm not able to see the app running on this node, but I can see from the service logs that it was previously - so I'm assuming you went ahead and turned it off!